src/overlays/actors/ovl_En_Door/z_en_door.c

```diff
@@ -113,15 +144,15 @@ void EnDoor_Init(Actor* thisx, PlayState* play2) {
     }
 
     // Double doors
-    if (this->actor.params & 0x40) {
+    if (ENDOOR_DOUBLE_DOOR(&this->actor)) {
```

Comment:
we havent agreed on a standardization for actor params yet, but I feel like it should be like `ENDOOR_GET_IS_DOUBLE_DOOR`. the is sounds kinda weird, but "get double door" is also weird. dunno

Reply:
in general i think the params getter macros should have GET in the name at least. but again this is something we will iron out as we standardize params macros

Reply:
`ENDOOR_IS_DOUBLE_DOOR` ? usually "is" is used for boolean getters

Reply:
i guess, not really a fan cause it doesnt sound to me like a params macro
guess ill worry about this later when params macros exist and have a more unified naming scheme

Reply:
Should others change as well like `ENDOOR_TYPE` -> `ENDOOR_GET_TYPE`
Can we revisit this once we agree on something (btw I thought we did agree on this, basically following `BgMizuMovebg`'s `MOVEBG_` style but passing `thisx`)

Reply:
I wouldnt call mizumovebg as an agreement no, its just what mzx decided to name it one time. But yea we can revisit it after tharos params changes are in and we actually start naming these

Reply:
Yeah I know it's old stuff but I mean, in my memories we agreed on basically what `BgMizuMovebg` does, but use `thisx` as argument instead of `params`

Reply:
yea we talked about things like what we pass to it, but nothing about a name (at least afaik). which is what my comment about this is